packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -444,11 +444,14 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
   }
 
   protected onInputBlur = (ev: React.FocusEvent<HTMLInputElement | Autofill>): void => {
-    // Only blur if an unrelated element gets focus. Otherwise treat it as though it still has focus.
+    if (this.props.inputProps && this.props.inputProps.onBlur) {
```

Comment:
Should this be in the conditional below?

Reply:
No, this is a callback specifically for the input, there could be a case where something should happen only explicitly when input is blurred. However, the picker should only act when it loses focus entirely.